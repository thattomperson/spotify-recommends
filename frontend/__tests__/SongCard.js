jest.mock('../store');
const SongCard = require('../SongCard.svelte')

describe('SongCard', () => {
  it('Should Render Correctly', () => {
    // SongCard
    
    let tree = {
      node: 'test'
    }

    expect(tree).toMatchSnapshot()
  })
})