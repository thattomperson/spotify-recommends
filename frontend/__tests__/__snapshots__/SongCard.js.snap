// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SongCard Should Render Correctly 1`] = `
Object {
  "node": "test",
}
`;
